lib/processor.spec.js

```diff
@@ -24,6 +24,7 @@ test.cb('processor.processFile - call entries with correct properties coming fro
           t.is(options.accessToken, 'bar')
           t.is(options.host, 'baz')
           t.is(options.space, 'foo')
+          t.is(option.environment, 'master')
```

Comment:
There is no value added with all these additions. You better can ask @johanb how to add specs for this. Probabbly you want to add a spec to prove using an another environment will return different data.